### ISSUE TITLE
Broken Plugin

### ISSUE THREAD

**Issue by Mantasesurvives** [Raziel7893/WindowsGSM.Smalland#1]
Hello, Would you mind double checking this plugin? I have installed it and it does not work. I have 12 other windowsGSM servers, and I have gotten all of them to work. It appears the server is running, Although it does not allow toggling the console on and off. When I launch the game using the batch file the plugin creates there is a console. Regardless of the commands I change, I cannot get the server to report to steam, when checking with steam query tool.

**Comment by Raziel7893** [comment]
uhm it works for me. maybe the start-bat changed with the v1 release and i can't read the necessarry keys for the server. will investigate in the next few hours. as temponary fix you can copy and paste the 3 set DEPLOYMENTID, CLIENTID and CLIENTSECRET  in smalland_conf.json ?

the steam querry tool never worked for me dueto no info available during Early access.

will look into that now.

**Comment by Raziel7893** [comment]
uhm no created a new server, it also works. (with embedded console)
you tried finding the server through your ingame serverbroser(like the readme suggests)? that's the only way i could found the server (via batch and via windowsGSM).  **This is also confirmed via googleing, the dedicated server connection is a bit tricky., and i can not find any upate about that**

after the server runs (takes a few minuteds the first start) i can find it in the ingame serverlist and connect
 (use the filter to reduce the list)
<img width="1044" alt="image" src="https://github.com/Raziel7893/WindowsGSM.Smalland/assets/159198554/ad65db2a-af67-4696-9604-a826bfc85812">

**Comment by Mantasesurvives** [comment]
Yes I tried in the ingame manager first, then when it did not find it I tried the querytool, and also the embedded console does not work on my server. Did you try this with the latest plugin? or the plugin you already had installed?

Okay, Things are better, I don't know embedded console works but toggle doesn't, but that is minor. I do see the server in the list but I cannot connect. What are the default ports? I see you have a reference to a port 8211 but I could not find any documentation as to the default ports from steam, or the website. 


Also, the server is not reporting the current server version (reports as 1.0 but current versiuon 2.xxx

**Comment by Mantasesurvives** [comment]
Update : Thank you for the help, I Have the server working now. One thing to note, If users can not connect to the server, they may have to use a VPN, on their normal game computer. I have seen this with one other server, (no one survived).

**Comment by Raziel7893** [comment]
hmm thats strange with the VPN, i never had that problem. @Mantasesurvives  and in the screenshot you see 7777 is the defaultport, the configuration is a one to one copy of the startscript, as there is absolutly no info at all besides the shipped start-server.bat script.

EDIT: what do you exactly mean with VPN? to where? to your server? 
EDIT2: as soon as there is any more info about the server query stuff i will add that. it's a known issue
